.github/workflows/crates-io.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish prusti-contracts to Crates.io
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [master, publish-crates-io]
+    paths: 'prusti-contracts/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
```

Comment:
Why? Python 3 [is pre-installed](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#language-and-runtime) on `ubuntu-*` images.